frontend/components/home/HAboutUs.tsx

```
@@ -0,0 +1,118 @@
+                                <span className='sub-title'>Get More About Us</span>
+                                <h2 className='title'>
+                                    Thousand Of Top{''}
+                                    <span className='relative mx-2'>
+                                        <svg
+                                            x='0px'
+                                            y='0px'
+                                            preserveAspectRatio='none'
+                                            viewBox='0 0 209 59'
+                                            fill='none'
+                                            xmlns='http://www.w3.org/2000/svg'
+                                        >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)